signer/src/testing/dummy.rs

```diff
@@ -510,6 +512,48 @@ impl fake::Dummy<fake::Faker> for SigHash {
     }
 }
 
+impl fake::Dummy<std::ops::Range<u64>> for BitcoinBlockHeight {
+    fn dummy_with_rng<R: rand::Rng + ?Sized>(config: &std::ops::Range<u64>, rng: &mut R) -> Self {
+        // Overwriting config because values greater then i64::MAX will cause db conversion errors.
+        let mut config = config.clone();
+        if config.end > i64::MAX as u64 || config.start >= config.end {
+            config.start = 0;
+            config.end = i64::MAX as u64;
+        }
+        rng.gen_range(config).into()
+    }
+}
+
+impl fake::Dummy<std::ops::Range<u64>> for StacksBlockHeight {
+    fn dummy_with_rng<R: rand::Rng + ?Sized>(config: &std::ops::Range<u64>, rng: &mut R) -> Self {
+        // Overwriting config because values greater then i64::MAX will cause db conversion errors.
+        let mut config = config.clone();
+        if config.end > i64::MAX as u64 || config.start >= config.end {
+            config.start = 0;
+            config.end = i64::MAX as u64;
+        }
+        rng.gen_range(config).into()
+    }
+}
+
+impl fake::Dummy<fake::Faker> for BitcoinBlockHeight {
+    fn dummy_with_rng<R: rand::Rng + ?Sized>(config: &fake::Faker, rng: &mut R) -> Self {
+        let x: u64 = config.fake_with_rng(rng);
+        let y: u64 = config.fake_with_rng(rng);
+        let range = std::cmp::min(x, y)..std::cmp::max(x, y);
+        range.fake_with_rng(rng)
```

Comment:
We should probably simplify this to `rng.gen_range(0..i64::MAX as u64).into()`. Same for `StacksBlockHeight`.

Reply:
This impl will not use `config`, and @cylewitruk and @matteojug highlighted that we want use this config... However, imo, we ok to just have uniform on 0..i64max for all random heights

Reply:
For this `impl` the config is the `fake::Faker` type, which is not really a config. It's a unit type, so it's okay to not use it in this cases.

Reply:
I think this [8b76743](https://github.com/stacks-sbtc/sbtc/pull/1448/commits/8b76743e5e1b1b3a0dd12c2cc8d327cd815fe74e) resolves this thread